hello.php

```diff
@@ -1,7 +1,17 @@
 <?php
 
-$string = "Hello %s\n";
+$hour = date('G');
 
-echo sprintf($string, $argv[1]);
+if ( $hour >= 5 && $hour <= 11 ) {
+    $greeting = "Good Morning";
```

Comment:
Did you consider using the Decorator pattern here? Here's a good article on it: https://sourcemaking.com/design_patterns/decorator/php